sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawContainer.java

```diff
@@ -15,7 +16,12 @@ private RawContainer(
             @JsonProperty("image-name") String imageName,
             @JsonProperty("rlimits") WriteOnceLinkedHashMap<String, RawRLimit> rlimits) {
         this.imageName = imageName;
-        this.rlimits = rlimits;
+
+        if (rlimits != null) {
```

Comment:
Makes sense. I agree about reducing the scope of a null field, just wanted to ensure consistency in one direction or another.